.github/workflows/build-vcpkg.yml

```diff
@@ -145,4 +204,4 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: tfs-${{ matrix.name }}-${{ matrix.buildtype }}-luajit=${{ matrix.luajit }}-${{ github.sha }}
-          path: ${{ github.workspace }}
```

Comment:
Please restore the end of file newline :pray:

Reply:
I'm not the only one who loves to leave a line at the end uwu